pkgs/by-name/fo/forgejo/generic.nix

```diff
@@ -1,9 +1,12 @@
 {
   lts ? false,
   version,
+  rev ? null,
+  srcOverride ? null,
```

Comment:
To achieve something similar with different intentions, I used this pattern in ffmpeg:

https://github.com/NixOS/nixpkgs/blob/5a24308c384447887fe59453dd23b5cb09079593/pkgs/development/libraries/ffmpeg/generic.nix#L3-L11

I find it quite a bit more idiomatic and simple.

Reply:
Oh cool I didn't know that one can reuse attributes directly in the function args attrset, I'll rewrite this like you did. 👍

Reply:
Ah, problem is that `fetchFromGitea` is not available up there, your `fetchgit` is a builtin.